scripts/create-one.sh

```diff
@@ -5,9 +5,9 @@ set -o pipefail
 
 readonly PROG_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 readonly ROOT_DIR="$(cd "${PROG_DIR}/.." && pwd)"
-readonly STACK_DIR="${ROOT_DIR}/stack-nodejs-16"
+readonly STACK_DIR="${STACK_DIR:-${ROOT_DIR}/stack}"
 readonly BIN_DIR="${ROOT_DIR}/.bin"
-readonly BUILD_DIR="${ROOT_DIR}/build-nodejs-16"
+readonly BUILD_DIR="${BUILD_DIR:-${ROOT_DIR}/build}"
```

Comment:
its not entirely clear to me why we need create-one.sh and create.sh, can you explain?